templates/serverpod_templates/modulename_server/test/unit/example_util_test.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:test/test.dart';
+import 'package:modulename_server/src/endpoints/module_endpoint.dart';
+
+void main() {
+  test(
+      'Given ModuleUtil '
+      'when calling buildGreeting '
+      'then returns greeting', () async {
```

Comment:
Suggestion: Include verification of the name in the description.

Same comment here, it's worth making sure these are correct and give a good impression.

Reply:
[0b363f1](https://github.com/serverpod/serverpod/pull/2882/commits/0b363f114f37af63e6437992ca3f7066f2904259)